core/_dev/_utils/fn-3-thumbnails.php

```diff
@@ -323,25 +323,37 @@ function czr_fn_get_placeholder_thumb( $_requested_size = 'thumb-standard' ) {
     $filter = apply_filters( 'czr_placeholder_thumb_filter', false );
     //make sure we did not lose the img_src
     if ( false == $_img_src )
-      $_img_src = czr_fn_get_theme_file_url( CZR_ASSETS_PREFIX . "/front/img/{$_requested_size}.png" );
+      $_img_src = czr_fn_get_theme_file_url( CZR_ASSETS_PREFIX . "front/img/{$_requested_size}.png" );
+
+    $thumb_to_sizes = array(
+      //thumb => width, height
+      'thumb-medium' => array( '520', '245' ),
+      'thumb-small' => array( '160', '160' ),
+      'thumb-standard' => array( '300', '300' )
+    );
 
     $attr = array(
       'class'             => 'czr-img-placeholder',
       'src'               => $_img_src,
       'alt'               => trim( strip_tags( get_the_title() ) ),
-      'data-czr-post-id'  => $_unique_id,   
+      'data-czr-post-id'  => $_unique_id,
+      //$_requested_size is always forced to have a value present in this array $_sizes = array( 'thumb-medium', 'thumb-small', 'thumb-standard' );
```

Comment:
OK, yes I see how this is done earlier in the code. no need to check for entry existence then.